src/platforms/web/runtime/components/transition-group.js

```diff
@@ -117,7 +119,16 @@ export default {
         const el: any = c.elm
         const s: any = el.style
         addTransitionClass(el, moveClass)
-        s.transform = s.WebkitTransform = s.transitionDuration = ''
+        s.transform = s.WebkitTransform = ''
+        const duration: any = toNumber(
+          isObject(c.data.transition.duration)
+          ? c.data.transition.duration.move
+          : c.data.transition.duration
+        )
+        if (process.env.NODE_ENV !== 'production' && duration != null) {
+          checkDuration(duration, 'move', c)
+        }
+        s.transitionDuration = duration ? `${duration}ms` : ''
```

Comment:
No need to override the value if `duration` isn't set. (`0` should be considered as set.)

Reply:
@Justineo 
Thank you for reviewing, but original code is overriding the value at line 120. IMO, we shouldn't change it unless there is a specific reason.

Reply:
Let's keep the current behavior then :)